instance/management/commands/instance_statistics_csv.py

```diff
@@ -98,6 +98,13 @@ def add_arguments(self, parser):
             default=None,
             help='Path to the output file of the new CSV. Leave blank to use stdout.'
         )
+        parser.add_argument(
```

Comment:
@spokerman12, the management command should be left as-is and the code [here](https://github.com/open-craft/opencraft/blob/master/reports/tasks.py) should be updated to do the necessary filtering.

Reply:
I don't understand, on this [comment](https://tasks.opencraft.com/browse/BB-3670?focusedCommentId=188674&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-188674) you said 

> [implement] a flag to the existing command that takes in a date range and filters the instances down to that

- Are domains always required, then?
- Should I remove the --created flag?
- Should I create another db_periodic_task or edit the current send_trial_instances_report? The name is confusing as it suggests I should edit it.

Reply:
@spokerman12, yes, I said what you have quoted above. But I feel that it is better to leave the command as-is and generic enough and update the periodic task instead to filter and pass only the instances created in the most recent completed month. Does this make sense or do you have any questions?

Reply:
Makes sense! I implemented the changes suggested plus a test.